src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -27,6 +26,17 @@ export default function CustomFieldInputSingleChoice(props) {
     fill="skip"
   />);
 
+  const defaultValue = () => {
```

Comment:
This doesn't have to be a function! Unnecessary functions can create more memory pressure!

```suggestion
  const defaultValue = value ? value.label : '';
```